Gemfile

```diff
@@ -44,5 +48,23 @@ end
 group :development do
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails'
+  gem 'faker'
```

Comment:
Just a reminder: if we use faker in our tests, we'll still need to add faker configuration to the rails_helper